### DISCUSSION TITLE
Add Italian as a Language

### DISCUSSION THREAD

**Discussion by zairro** [GitbookIO/community#407] [Feature Requests]
Hello, 
I see that GitBook offers Spanish and a few other language options in the language section. I'd like to make a request for Italian to be added too. In general, it would be nice to see additional languages given that we have a large international audience.

**Comment by MadeByPP** [comment]
After contacting gitbook support, they responded that they currently have no plans to add support for other languages.

**Comment by aikoii** [comment]
Hi @zairro, thank you for your suggestion. I appreciate your input and would love to see you join similar discussions or vote on the topic here #291. (will be closing this in favour of 291)

Unfortunately, we don't have any immediate plans to add a new language (as @MadeByPP mentioned), but we may offer a way to do this in the future 🤞.
